docs/getting-started/datapackage-find-prepare-share-guide.md

```diff
@@ -4,59 +4,56 @@ date: 2018-04-26
 authors: Dmytro Gierman, Branko Djordjevic
 ---
 
-**In this guide we describe the way to gather the information and to make it presentable and easy to use for your customers.**
+**In this guide, we describe a way to gather information and make it presentable and easy to use for your customers.**
+
+Over time we've seen an increase in the adoption of the internet, with so much information available to us it can quickly become overwhelming. With this enormous data, numerous opportunities exist for those that can make sense of these datasets and present them in a clear and succinct manner.
```

Comment:
@lauragift21 i said we wanted to completely redo this intro. This stuff did not really make sense. Let's have a simple intro that explains what this tutorial does ...

Reply:
Noted @rufuspollock I'll work on these.